library/std/math.qs

```diff
@@ -85,36 +126,52 @@ namespace Microsoft.Quantum.Math {
         a < b ? a | b
     }
 
-    //
-    // Trigonometric functions
-    //
-
     /// # Summary
-    /// Represents the ratio of the circumference of a circle to its diameter.
+    /// Given an array of integers, returns the largest element.
     ///
-    /// # Ouptut
-    /// A double-precision approximation of the the circumference of a circle
-    /// to its diameter, π ≈ 3.14159265358979323846.
+    /// # Input
+    /// ## values
+    /// An array to take the maximum of.
     ///
-    /// # See Also
-    /// - Microsoft.Quantum.Math.E
-    function PI() : Double {
-        3.14159265358979323846
+    /// # Output
+    /// The largest element of `values`.
+    function Max (values : Int[]) : Int {
+        Fact(Length(values) > 0, "Array must contain at least one element.");
+        mutable max = values[0];
+        for idx in 1 .. Length(values) - 1 {
+            if values[idx] > max {
+                set max = values[idx];
+            }
+        }
```

Comment:
```suggestion
        for element in values[1...] {
            if element > max {
                set max = element;
            }
        }
```

Reply:
This can be done without the need to use indexes through slices.

Reply:
Neat!